packages/next/src/client/components/router-reducer/reducers/navigate-reducer.test.tsx

```diff
@@ -274,37 +274,7 @@ describe('navigateReducer', () => {
         "nextUrl": "/linking/about",
         "prefetchCache": Map {
           "/linking/about" => {
-            "data": Promise {
-              "status": "fulfilled",
-              "value": [
-                [
-                  [
-                    "children",
-                    "linking",
-                    "children",
-                    "about",
-                    [
-                      "about",
-                      {
-                        "children": [
-                          "__PAGE__",
-                          {},
-                        ],
-                      },
-                    ],
-                    <h1>
-                      About Page!
-                    </h1>,
-                    <React.Fragment>
-                      <title>
-                        About page!
-                      </title>
-                    </React.Fragment>,
-                  ],
-                ],
-                undefined,
-              ],
-            },
+            "data": Promise {},
```

Comment:
@ztanner These snapshot tests changed because the promise no longer contains a `value` property so the Jest can't serialize it. I read through the affected tests and the value of prefetch cache didn't seem relevant but let me know if there's a case where the lost test coverage is important. We shouldn't be changing implementation details solely to expose extra information in unit tests but I can add back the coverage in some other way where needed.